frontend/client-list/client-list.component.tsx

```diff
@@ -255,17 +256,27 @@ export type ClientListClient = {
   };
 };
 
-const initialState = {
-  status: ApiStateStatus.fetched,
-  apiData: {
-    pagination: {
-      currentPage: 0,
-      numPages: 0,
-      numClients: 0,
-      pageSize: 0
+function getInitialState(): ApiState {
```

Comment:
The changes here make it so that the initial `page` and `search` for ClientList is based on the URL params, instead of `null`